src/sinol_make/commands/inwer/__init__.py

```diff
@@ -0,0 +1,115 @@
+import argparse
+import os
+import multiprocessing as mp
+
+from sinol_make import util
+from sinol_make.commands.inwer.structs import TestResult, InwerExecution, VerificationResult, TableData
+from sinol_make.helpers import package_util
+from sinol_make.helpers.parsers import add_compilation_arguments
+from sinol_make.interfaces.BaseCommand import BaseCommand
+from sinol_make.commands.inwer import inwer_util
+
+
+class Command(BaseCommand):
+    """
+    Class for "inwer" command.
+    """
+
+    def get_name(self):
+        return "inwer"
+
+    def configure_subparser(self, subparser: argparse.ArgumentParser):
+        parser = subparser.add_parser(
+            self.get_name(),
+            help='Verify if input files are correct',
+            description='Verify if input files are correct'
+        )
+
+        parser.add_argument('inwer', type=str, nargs='?',
+                            help='path to inwer executable')
```

Comment:
Doesn't this have to be `--inwer`, just like the arguments below? Also, it might be better to name it `inwer_path`, so that it is immediately clear what this argument does.

Reply:
This way it's a positional argument, so the usage would be like this: `sinol-make run inwer prog/inwer2.cpp`. But I agree that the name should be different.

Reply:
Changed the name of this argument feacfe5b4ec4e81912c702878194fd353942f462

Reply:
> This way it's a positional argument

I see. Though when it's the only positional argument (especially that there are none such arguments in `run`), it seems inconsistent. Now the user needs to know that there are two kinds of arguments. With the big difference between the count of positional and non-positional, it might be better to have only non-positional arguments, thanks to which the user won't feel that "something is weirdly different", just like I did.